src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -103,6 +102,11 @@ object Coroutine {
     var $yield: Y = null.asInstanceOf[Y]
     var $result: R = null.asInstanceOf[R]
 
+    /**
```

Comment:
Take a look at the doc comments section here:

http://storm-enroute.com/dev/styleguide/

We have to indent as shown there, and use ticks for inline code, `{{{` and `}}}` for multiline code.